components/SpaceFact.js

```diff
@@ -0,0 +1,118 @@
+      details: 'Just as colors are made more dramatic in sunsets on Earth, sunsets on Mars, according to NASA,  would appear bluish to human observers watching from the red planet. Fine dust makes the blue near the Suns part of the sky much more visibilke, while normal daylight makes the Red Planets familiar rusty dust color the most perceptible to the human eye.'
+    },
+    {
+      fact: 'THERE ARE MORE STARS IN THE UNIVERSE THAN GRAINS OF SANDS ON EARTH',
+      details: 'The universe extends far beyond our own galaxy, The Milky Way, which is why scientists can only estimate how many stars are in space. However, scientists estimate the universe contains approximately 1,000,000,000,000,000,000,000,000 stars, or a septillion. While no one can actually count every single grain of sand on the earth, the estimated total is somewhere around seven quintillion, five hundred quadrillion grains.'
+   },
+    {
+      fact: 'ONE DAY ON VENUS IS LONGER THAN ONE YEAR',
+      details: 'Venus has a slow axis rotation which takes 243 Earth days to complete its day. The orbit of Venus around the Sun is 225 Earth days, making a year on Venus 18 days less than a day on Venus.'
+    },
+    {
+      fact: 'THERE IS A PLANET MADE OF DIAMONDS',
+      details: 'There’s a planet made of diamonds twice the size of earth The "super earth," aka 55 Cancri e, is most likely covered in graphite and diamond. Paying a visit to that planet would probably pay for the $12 million dollar space suit needed to get there!'
+    },
+  ]
+
+  const [spacefact, setSpacefact] = useState({})
+
+  const getSpacefact = () => {
+    const theSpacefact = SpaceFactArray[Math.floor(Math.random() * SpaceFactArray.length)]
+    setSpacefact(theSpacefact)
+  }
+
+  return (
+     <Container>
+        <FactButton
+        key={spacefact.fact}>
+          <FactText>{spacefact.fact}</FactText>
+          <DetailsText>{spacefact.details}</DetailsText>
+        </FactButton>
+        <ClickButton onPress={() => { getSpacefact()}}>
+          <ButtonTopText>PRESS</ButtonTopText>
+          <ButtonText>to learn cool things about space!</ButtonText>
+        </ClickButton>
+    </Container>
+  )
+}
+
+const Container = styled.View`
+margin: 0 auto;
+`
+const FactButton= styled.TouchableOpacity`
+justify-content: center;
+align-items: center;
+text-align: center;
+max-width: 240px;
+margin-bottom: 5px;
+`
+const FactText = styled.Text`
+color: pink;
+font-size: 18px;
+`
+const DetailsText = styled.Text`
+color: white;
+font-size: 16px;
+`
+const ClickButton = styled.TouchableOpacity`
+justify-content: center;
+align-items: center;
+text-align: center;
+background-color: #0A97B0;
+padding-top: 30px;
+padding-bottom: 10px;
+border: 2px solid #fff;
+width: 240px;
+height: 100px;
+border-radius: 8px;
+`
+const ButtonTopText = styled.Text`
+font-weight: 900;
+font-size: 18px;
+color: #fff;
+`
+const ButtonText = styled.Text`
+font-weight: 700;
+font-size: 14px;
+padding: 0 5px 0 5px;
```

Comment:
I think you have a really good foundation to complet this project. Your code is easy to follow and your nameing of the component is really good! I see that you have the same arry in two places,so just kepp one(I think you allready know that do, hehe :) )!Good work Caroline, Share the completed project when your done, i think is going to be awesome! :D